ee-9/feature-pack/pom.xml

```diff
@@ -192,6 +192,9 @@
                             <resources>
                                 <resource>
                                     <directory>${mp.galleon.common.resources.directory}</directory>
+                                    <excludes>
+                                        <exclude>layers/standalone/jaxrs</exclude>
+                                    </excludes>
```

Comment:
Is this needed? The next execution should copy over this fine with the one from the local src/main/resources.